contrib/guix/INSTALL.md

```diff
@@ -72,11 +72,11 @@ writing (July 2021). Guix is expected to be more widely packaged over time. For
 an up-to-date view on Guix's package status/version across distros, please see:
 https://repology.org/project/guix/versions
 
-### Debian 11 (Bullseye)/Ubuntu 21.04 (Hirsute Hippo)
+### Debian 11 (Bullseye)/Ubuntu 22.04 (Jammy Jellyfish)
```

Comment:
```suggestion
### Debian / Ubuntu
```

Probably easiest to just drop the versioning / names from here.

Reply:
would second this

Reply:
Thank you for the code suggestion! I just committed it and GitHub auto resolved this comment.